.gitlab-ci.yml

```diff
@@ -12,6 +12,14 @@ test:integration-test-for-p2p-network:
     - sudo python3.6 -m pip install argparse docker pexpect requests
     - sudo ./scripts/p2p-test-tool.py -b -p 1
 
+test:integration-test-for-artifact-creation:
+  stage: test 
+  tags:
+    - preloaded-ubuntu1804 
+  script:
+    - ./scripts/install_bnfc.sh
+    - sudo sbt -Dsbt.log.noformat=true clean rholang/bnfc:generate node/rpm:packageBin node/debian:packageBin node/universal:packageZipTarball 
```

Comment:
I know this is a little long of a line but multi-line is a little ugly in the gitlab-ci yaml config. I still think this is easier to read better than putting in some outside script where I could do multi-line